src/components/tl/mlx5/tl_mlx5_pd.c

```diff
@@ -125,9 +125,9 @@ ucc_status_t ucc_tl_mlx5_socket_init(ucc_tl_mlx5_context_t *ctx,
     return UCC_ERR_NO_MESSAGE;
 }
 
-static ucc_status_t client_recv_data(int *shared_cmd_fd,
-                                     uint32_t *shared_pd_handle,
-                                     const char *sock_path,
+static ucc_status_t client_recv_data(int        *shared_cmd_fd,
+                                     uint32_t   *shared_pd_handle,
+                                     const char *sock_path, int *sock_p,
```

Comment:
pls don't align like this, see here https://github.com/openucx/ucc/blob/master/docs/codestyle.md